rdt-open-reader/src/covidIndonesia/res/layout/activity_covid_patient_history.xml

```diff
@@ -1,51 +1,68 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto">
+        xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <androidx.core.widget.NestedScrollView
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:fillViewport="true">
-        <LinearLayout
-            android:layout_height="match_parent"
+    <LinearLayout
             android:layout_width="match_parent"
+            android:layout_height="match_parent"
             android:orientation="vertical">
 
-            <include
-                android:id="@+id/patient_symptoms_history_section"
-                layout="@layout/patient_history_section"
+        <androidx.core.widget.NestedScrollView
```

Comment:
Why not add the date within this `NestedScrollView`?

Reply:
Ellya asked to make it fixed.

Reply:
What issue is being addressed by the change?

Reply:
I mean "fixed layout", not scrolling when you scroll on scrollview. :D

Reply:
like the date should be fixed (not move when scrolling)?

Reply:
Yes.